server/views/novels-main.pug

```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```

Comment:
Looks like some stuff left in from a merge conflict :) And one at the bottom of the file too

Reply:
This is cleaned now.